src/app/components/Table/TableCell/TableCell.module.scss

```diff
@@ -2,7 +2,7 @@
   display: flex;
   flex-direction: column;
   justify-content: center;
-  align-items: center;
+  align-items: flex-start;
```

Comment:
I'm not sure

Reply:
Yes, I'm sure. I want to place the content exactly on the left edge of the block